ga4gh/repo_manager.py

```diff
@@ -0,0 +1,132 @@
+"""
+Manages a GA4GH data repository
+"""
+import os
+
+
+class RepoManagerException(Exception):
+    pass
+
+
+class RepoManager(object):
+
+    datasetsDirName = "datasets"
+    referenceSetsDirName = "referenceSets"
+    readsDirName = "reads"
+    variantsDirName = "variants"
+
+    def __init__(self, repoPath):
+        self.repoPath = repoPath
```

Comment:
Should make these protected variables, following the usual conventions (``self._repoPath``), etc.